app/serializers/comment_serializer.rb

```diff
@@ -0,0 +1,26 @@
+class CommentSerializer
+  def initialize(comment)
+    @comment = comment
+  end
+
+  def to_h
+    {
+      content: comment.content,
+      user_image_url: user.image_url,
+      user_name: user.name,
+      created_at: comment.created_at.to_i
+    }
+  end
+
+  private
+
+  attr_reader :comment
+
+  def user
+    comment.user || anonymous_user
```

Comment:
after unifying comment model you could move it there. then you won't need an empty user method anymore